CONTRIBUTING.md

```diff
@@ -41,7 +41,7 @@ $ npm run start
 
 ## Pull Requests
 
-We actively welcome your pull requests.
+We actively welcome your pull requests. Each pull request must be approved by 2 contributors (not including yourself).
```

Comment:
I'm thinking we should add this under a separate section specifically for maintainers, as it won't apply to the majority of folks submitting PRs.

@calavera thoughts?